x/leverage/types/proposal.go

```diff
@@ -4,23 +4,32 @@ import (
 	gov "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
 )
 
-var proposalTypeMsgGovUpdateRegistry = MsgGovUpdateRegistry{}.Type()
+var (
+	proposalTypeMsgGovUpdateRegistry          = MsgGovUpdateRegistry{}.Type()
+	proposalTypeMsgGovUpdateSpecialAssetPairs = MsgGovUpdateSpecialAssetPairs{}.Type()
+)
 
 func init() {
 	gov.RegisterProposalType(proposalTypeMsgGovUpdateRegistry)
+	gov.RegisterProposalType(proposalTypeMsgGovUpdateSpecialAssetPairs)
 }
 
 // Implements Proposal Interface
-var _ gov.Content = &MsgGovUpdateRegistry{}
-
-// GetTitle returns the title of a community pool spend proposal.
-func (msg *MsgGovUpdateRegistry) GetTitle() string { return msg.Title }
+var (
+	_ gov.Content = &MsgGovUpdateRegistry{}
+	_ gov.Content = &MsgGovUpdateSpecialAssetPairs{}
+)
```

Comment:
I noticed we have this for leverage's gov message but not ugov or incentive. Does this mean I can skip adding the content of this file for `MsgGovUpdateSpecialAssetPairs`?